code/datums/components/grillable.dm

```diff
@@ -82,6 +81,9 @@
 		if(original_object.custom_materials)
 			grilled_result.set_custom_materials(original_object.custom_materials)
 
+	if(IS_EDIBLE(grilled_result))
+		SSblackbox.record_feedback("tally", "food_made", 1, grilled_result.type)
```

Comment:
Can we move this to a proc or to a define since we use it in two places?

Reply:
I'm not sure what you're asking for here. You want the `if(IS_EDIBLE)` and blackbox logging to be in a proc/define? If so, I don't really see the point in making a proc for two reasonably readable lines of code that're used in two places.

Reply:
Eh at the very least can you make "food_made" a define, my concern here is just the two drifting

Reply:
Gotcha, I can do that